pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -859,6 +859,48 @@ func getSRIOVPCIAddresses(ifaces []v1.Interface) map[string][]string {
 	return networkToAddressesMap
 }
 
+// This function parses the GPU-PASSTHROUGH-DEVICES variable that is set by
+// device plugin. It lists PCI IDs for devices allocated to the pod. The format
+// is as follows:
+// "": for no allocated devices
+// "0000:81:11.1,": for a single device
+// "0000:81:11.1,0000:81:11.2[,...]": for multiple devices
+func getGpuAddresses() []string {
+	addrString, isSet := os.LookupEnv("GPU-PASSTHROUGH-DEVICES")
```

Comment:
I wonder if we shouldn't make this variable generic even more. I think there is nothing significant in regular GPU passthrough as we attach it as a regular PCI device.
Perhaps we should make this variable available for all generic PCI devices. This way if we will want to passthrough an arbitrary PCI device any device plugin could add it to this list.

Reply:
@booxter @fabiand @roman what do you think?

Reply:
If we decide to make this env variable generic, we will also have to make the API more generic. Current the API has `vmi.Spec.Domain.Devices.Gpus` which is serving both GPUs and VGPUs. We might need to have  `vmi.Spec.Domain.Devices.Pci` and  `vmi.Spec.Domain.Devices.Mediated`

Reply:
@vladikr AFAIU this variable name is specific to DP. While having a generic way to pass PCI devices may be useful, particular device types - whether it's SR-IOV NIC VF or vGPU - may require additional handling code before passing them to libvirtd, in which case having specific API could be handy. There's probably space for a generic PCI DP that would standardize the interface and handle devices in a generic way, but that would be a separate independent project that we integrate with. AFAIU it doesn't exist right now.

Reply:
@booxter This PR use two variables to pass a list of PCI devices and mdev UUIDS (GPU-PASSTHROUGH-DEVICES and VGPU-PASSTHROUGH-DEVICES) to KubeVirt.
I'm specifically talking about GPU-PASSTHROUGH-DEVICES, which contains a list of PCI devices that don't require any specific configuration. `createHostDevicesFromPCIAddresses` can handle any generic PCI device in the same manner.

This PR defines an interface between the DP(s) and Kubevirt. All DPs that will provide vGPUs to kubevirt will use the VGPU-PASSTHROUGH-DEVICES variable.
As `GPU-PASSTHROUGH-DEVICES` is not significant to GPU only, I suggest to rename it to now to something that any generic PCI DP will use.

Reply:
@fabian and @roman What do you think about @vladikr suggestion about the env variable and how also any suggestions on how should the API look like?
@vladikr Pci device passthrough libvirt xml can also have element `rom bar`. GPU passthrough does not need it. So is GPU passthrough really a generic PCI device passthrough?